mmv1/products/identityplatform/Config.yaml

```diff
@@ -59,3 +59,70 @@ properties:
     name: 'autodeleteAnonymousUsers'
     description: |
       Whether anonymous users will be auto-deleted after a period of 30 days
+  - !ruby/object:Api::Type::NestedObject
+    name: 'blockingFunctions'
+    description: |
+      Configuration related to blocking functions.
+    properties:
+      - !ruby/object:Api::Type::Map
+        name: 'triggers'
+        required: true
+        description: |
+          Map of Trigger to event type. Key should be one of the supported event types: "beforeCreate", "beforeSignIn".
+        key_name: event_type
+        value_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: 'functionUri'
+              required: true
+              description: |
+                HTTP URI trigger for the Cloud Function.
+            - !ruby/object:Api::Type::Time
+              name: 'updateTime'
+              output: true
+              description: |
+                When the trigger was changed.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'forwardInboundCredentials'
+        description: |
+          The user credentials to include in the JWT payload that is sent to the registered Blocking Functions.
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'idToken'
+            description: |
+              Whether to pass the user's OIDC identity provider's ID token.
+          - !ruby/object:Api::Type::Boolean
+            name: 'accessToken'
+            description: |
+              Whether to pass the user's OAuth identity provider's access token.
+          - !ruby/object:Api::Type::Boolean
+            name: 'refreshToken'
+            description: |
+              Whether to pass the user's OAuth identity provider's refresh token.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'quota'
+    description: |
+      Configuration related to quotas.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: 'signUpQuotaConfig'
+        description: |
+          Quota for the Signup endpoint, if overwritten. Signup quota is measured in sign ups per project per hour per IP.
+        properties:
+          - !ruby/object:Api::Type::Integer
+            name: 'quota'
+            description: |
+              A sign up APIs quota that customers can override temporarily.
+          - !ruby/object:Api::Type::Time
+            name: 'startTime'
+            description: |
+              When this quota will take affect.
+          - !ruby/object:Api::Type::String
+            name: 'quotaDuration'
+            description: |
```

Comment:
Should we also add an example of how to structure the duration string? E.g. https://github.com/GoogleCloudPlatform/magic-modules/blob/31547b00e1e93da8fd198d133a17df4264d4eb4d/mmv1/products/alloydb/Cluster.yaml#L171